### PR TITLE
feat: Implemented RelaxFeedback Settings

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #17 partly.

**Changes**: Implemented RelaxFeedback Settings.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
@CloudyPadmal @iamareebjamal @CoderVishalSehgal Please review, All co-developers are welcome to review. Cheers!

**Comment by jddeep** [comment]
@iamareebjamal Please have a look. I had requested for peer reviews as well.

**Comment by vishal-sehgal** [comment]
@jddeep Will check it out and let you know.

**Comment by jddeep** [comment]
> @jddeep Will check it out and let you know.

@CoderVishalSehgal Any updates? Did you check it out?

**Comment by jddeep** [comment]
@iamareebjamal Please have a look.

File: app/src/main/java/io/neurolab/model/RelaxFeedbackSettings.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    public double trainingFactor = 0.1;
+
+    @Override
+    public String getFeedbackSettingsName()
+    {
+        return "relax";
+    }
+
+    public RelaxFeedbackSettings(DefaultFFTData fftData, ReentrantLock lock, Config config)
+    {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/model/RelaxFeedbackSettings.java]: This is Java, not C. Use Java style guide

**Comment by jddeep** [comment]
@iamareebjamal Made the changes. Please review and merge.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**